core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -2486,10 +2486,16 @@ public FileSystemCommand workerHeartbeat(long workerId, List<Long> persistedFile
     boolean ownerGroupChanged = false;
     boolean permissionChanged = false;
     if (options.getOwner() != null) {
+      if (options.getOwner().isEmpty()) {
+        throw new AccessControlException("It is not allowed to set owner to empty");
+      }
       inode.setOwner(options.getOwner());
       ownerGroupChanged = true;
     }
     if (options.getGroup() != null) {
+      if (options.getGroup().isEmpty()) {
+        throw new AccessControlException("It is not allowed to set group to empty");
+      }
```

Comment:
Good point.
- Checking on the client side can make it fail faster, but would still require server-side check because an old-version client can `chown`.
- Checking on the server side option `alluxio.master.file.options.SetAttributeOptions` is feasible but that would require some validation method to check non-empty owner/group when creating from `SetAttributeTOptions` and set fields. Do you prefer the latter approach?